scripts/publish-to-npm.sh

```diff
@@ -56,6 +57,14 @@ fi
 git add packages/docs/
 git commit -m "docs for version ${VERSION_PREFIX}"
 
+# copy contracts
+rm -rf packages/generated/deployments
+cp -r contracts/deployments packages/generated/deployments
```

Comment:
how about `packages/generated/dev` for abis?

Reply:
that should be part of the normal ci build - are those out of sync?